nuxeo-java-client-test/src/main/resources/package.xml

```diff
@@ -9,7 +9,6 @@
   <uninstaller restart="true" />
   <nuxeo-validation>nuxeo_certified</nuxeo-validation>
   <platforms>
-    <platform>server-10.10*</platform>
     <platform>lts-2021.*</platform>
     <platform>lts-2023.*</platform>
   </platforms>
```

Comment:
Maybe worth updating it to a range? 2021+

Reply:
I would say no, because the package needs work to be compatible with a new LTS.

Reply:
At least
```
<target-platform>
    <name>lts</name>
    <version>[2021.0,2025.0)</version>
  </target-platform>
```
?